lib/3scale/api/http_client.rb

```diff
@@ -81,6 +81,13 @@ def debug?
         ENV.fetch('3SCALE_DEBUG', '0') == '1'
       end
 
+      # Helper to build a query string
+      def build_qs(path, params)
+        path = "#{path}.#{format}"
+        path << "?#{params.map { |k, v| "#{k}=#{v}" }.join('&')}" unless params.nil?
```

Comment:
This should be just `URI.encode_www_form`.
And the method name is not really obvious because it is not building QS, but building it and appending it to the path.